wct.conf.js

```diff
@@ -13,14 +13,14 @@ module.exports = {
         {
           deviceName: 'Android GoogleAPI Emulator',
           platformName: 'Android',
-          platformVersion: '10.0',
+          platformVersion: '11.0',
           browserName: 'chrome',
           browserVersion: 'latest'
         },
         {
           deviceName: 'Android GoogleAPI Emulator',
           platformName: 'Android',
-          platformVersion: '9.0',
+          platformVersion: '10.0',
```

Comment:
Sauce Labs has sent an email claiming that ACTION REQUIRED for the vaadin-router testing config. This may be relevant here:
> Hello,
>
> Sauce Labs is scheduling to update Android 10 to a newer browser version of Chrome on Monday, February 22nd, 2021. The update will take Chrome from version 74 to version 86. Testing on this version of Chrome and all subsequent versions will primarily support the W3C WebDriver protocol.
>
> In order to continue testing with Android 10, users are REQUIRED to make changes to their WebDriver capabilities. To prepare for this update, we recommend that you create a new version of your test scripts that include the W3C capabilities and on February 22nd, replace your existing script with this updated version.
>
> To ensure that you do not have any downtime, please update your current tests. If you are unable to update your tests to be W3C compliant yet, please review our example below for setting W3C capabilities to false. For help, view the W3C documentation.
>
> Code Example:
>
> Before (JSONWP):
> ```json
> {
>       "browserName": "chrome",
>       "deviceName": "Android GoogleAPI Emulator",
>       "platformName": "Android",
>       "platformVersion": "10.0",
>       "automationName": "UiAutomator2",
>       "appiumVersion": "1.18.1",
>       "name": "Appium JSONWP Test"
> },
> ```
>
> After (W3C):
> ```json
> {
>       "browserName": "chrome",
>       "appium:deviceName": "Android GoogleAPI Emulator",
>       "platformName": "Android",
>       "appium:platformVersion": "10.0",
>       "automationName": "UiAutomator2",
>       "sauce:options": {
>             "appiumVersion": "1.18.1",
>             "name": "Appium W3C Test"
>       }
> }
> ```
>
> Or, IF you are unable to enable W3C by then and still want to use the old JSONWP in combination with the latest version of ChromeDriver you can add the following to your capabilities below. Please note that when Selenium 4 is released the chance will be very high that browser-vendors will remove the old JSONWP from their drivers which eventually will break your tests.
> ```json
> {
>       "browserName": "chrome",
>       "deviceName": "Android GoogleAPI Emulator",
>       "platformName": "Android",
>       "platformVersion": "10.0",
>       "automationName": "UiAutomator2",
>       "appiumVersion": "1.18.1",
>       "name": "Appium JSONWP W3C False Test"
>       "goog:chromeOptions": {
>             "w3c": false
>       }
> },
> ```
>
> In addition, we plan to update the default version of Appium for Android 10 from 1.8.0 to 1.18.0. This will unlock features and capabilities available on the newer version. If Appium 1.8.0 is still required for testing, it can be specified within the capabilities. We will release a blog update when the update is live to minimize the impact this may have on your work.
>
> If you have questions or need assistance, please contact our Support team at support@saucelabs.com.
>
> Happy Testing!
>
> Sauce Labs

Reply:
Thanks. The way I read it is we can't move to the new W3C Test yet but should do it on/after Feb 22nd. I think we can keep this part as is for now and when update it later when/if it breaks. But good that you noticed this, now we're aware that this change is coming and not surprised if this test breaks later this month. :)